docs/commands/join.md

```diff
@@ -1,59 +1,68 @@
+---
+tocMaxDepth: 3
+---
 # `join`
 
-<div class="warning"><b>Important</b>
+## Introduction
 
+:::warning Important
 The `join` command is considered an experimental feature. This means it's still a work in progress and may go through major changes.
-</div>
 
+The `join` command supports OpenAPI 3.x definitions only.
+:::
 
-Maintainers of multiple API definitions can benefit from storing each endpoint as a standalone API definition file. However, this approach is not supported by the majority of OpenAPI tools, as they require a single API definition file.
+With the `join` command, Redocly OpenAPI CLI can combine two or more API definition files into one so that maintainers of multiple API definitions can benefit from storing each endpoint as a standalone API definition file. However, this approach is not supported by the majority of OpenAPI tools, as they require a single API definition file.
```

Comment:
These two sentences don't connect logically. In the original text, the meaning was something like this:

1) Maintainers sometimes split their endpoints into standalone files instead of keeping them in the root file.
2) Most OpenAPI tools don't work with standalone files like that.
3) The join command helps with this problem because it can combine those standalone files into one. 

In this new version, these things are out of order. The second sentence talks about the approach not being supported because other tools need a single file...but you already explained how to get the single file in the previous sentence? So this second sentence is just confusing.

You may want to rewrite this part so that you start with the problem first, and then explain how the join command can resolve it. Or you can start by describing what the join command does, and then explain in which use-cases it can be the most helpful.